### PR TITLE
Wip master

### PR DESCRIPTION
**Pull request by azat-co**
CC @AnnJohn @vicfriedman

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,26 @@
-# node-hello-world-lab
\ No newline at end of file
+# Node Hello World
+
+## Objectives
+
+1. Implement Hello World in Node
```
Comment (victhevenot) [inline README.md:5]: Numbers 1 and 2

File: test/index.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
+var test = require('tape'),
```
Comment (victhevenot) [inline test/index.js]: these tests aren't particularly readable for a student to look at and understand how this works. They are familiar with Jasmine - will that testing framework work here? or is there a way to refactor or add comments to simplify?

**Comment by victhevenot** [comment]
don't know if you'd finished this yet, but the test is failing on the wip-solution branch. im getting the following error:

```
module.js:340
    throw err;
          ^
Error: Cannot find module 'tape'
```

**Comment by victhevenot** [comment]
Hey Azat, Thanks for re-writing the tests in Mocha. 

I took a look a look at the spec, and had a hard time understanding each line of code. I'm not super familiar with child processes, etc. With that in mind, our students are going to have a really hard time reading those tests. There is a difference in tests for a student who is learning to code versus tests for product code. We want our tests to be as readable and clear for our students as possible.

I played around and came up with the following:

In a solution in hello_world.js:

```
exports.sayHey = function(){
  return "Hello World";
}
```

And changed the test to look like this:

```
var expect = require('chai').expect;
var person = require('../hello_world');

describe('node sayHey', function () {
  it('must return Hello World', function(done){
    expect(person.sayHey()).to.equal('Hello World');
    done();
  })
})
```

we'd have to change the instructions to let students know they need to define a function expression that returns "Hello World" and that the variable storing the function needs to have `exports.` directly in front for the tests to run.

And also explain that we have to require a file and store it in a variable, and then call any functions defined in that file by calling it on the variable. but I don't think those things will be hard to grasp.

It seems like your tests we're designed for console.log, but at the complexity of the tests, it might be better to just have them return a value from a function. especially since this is their first experience running their own mocha tests.

**Comment by azat-co** [comment]
updated according to the feedback (modular tests) CC @AnnJohn @vicfriedman

**Comment by azat-co** [comment]
checked out the readme from wip-solution CC @vicfriedman

**Pull request merged**